imsv-docs-docusaurus/openapi/endpoints/funding-sources/models/create-withdrawal-intent.yaml

```diff
@@ -0,0 +1,80 @@
+type: object
+properties:
+  amount:
+    type: number
+    description: Amount to withdraw.
+    example: 300
+  id:
+    type: string
+    description: ID of the Withdrawal Intent.
+    example: 91ad6fea3b52ca58d60d7fd310f789ec
+  createdAt:
+    type: string
+    description: Creation date of the Withdrawal Intent.
+    example: "2023-08-14T12:32:54"
+  expiresAt:
+    type: string
+    description: Expiration date of the Withdrawal Intent.
+    example: "2023-08-14T12:32:54"
+  walletAddress:
+    type: string
+    description: The wallet address to which the funds will be sent.
+    example: "0xA3058369d6A481B1ff08F62B352409c3D709De9b"
+  execution:
+    type: object
+    properties:
+      contractAddress:
+        type: string
+        description: The address of smart contract to invoke.
+        example: '0xEe076427B04DDD26729889C869d3aE524A6362fD'
+      method:
+        type: string
+        description: The contract function to call.
+        example: 'withdraw'
+      abi:
+        description: The JSON ABI of the smart contract (contains only required details. more details here https://docs.soliditylang.org/en/v0.8.19/abi-spec.html#json)
```

Comment:
we're only returning the singular method with the abi, is this the right description @javcode ?

Reply:
yeah the description is a bit misleading. We should clarify that this the abi just for the function being executed. But I'm not sure how this would work for other chains so I guess it should be fine